core/Piwik.php

```diff
@@ -176,7 +176,7 @@ public static function getRandomTitle()
     public static function getCurrentUserEmail()
     {
         $user = APIUsersManager::getInstance()->getUser(Piwik::getCurrentUserLogin());
-        return $user['email'];
```

Comment:
I think the phpdocs need to be changed here too. And do we need to mention in the developer changelog that the value can be empty now? I guess it might always have been possible if the user was `anonymous`...

Reply:
The phpdocs now actually would fit. Before it could maybe have returned `null` if the user array was empty.
Don't think we need to mention it in the developer docs. The change actually only avoids a possible warning in this case.